.devcontainer/development/Dockerfile

```
@@ -52,7 +52,7 @@ RUN curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | d
     && apt-get install -y gh
 
 # Install Go
-RUN apt-get update && apt-get install -y golang-go
+RUN apt-get update || true && apt-get install -y --fix-missing golang-go
```

Hadolint (2.14.0)

[info] Avoid additional packages by specifying `--no-install-recommends`

(DL3015)